src/vstart.sh

```diff
@@ -900,7 +900,6 @@ $CCLIENTDEBUG
         ; needed for s3tests
         rgw crypt s3 kms backend = testing
         rgw crypt s3 kms encryption keys = testkey-1=YmluCmJvb3N0CmJvb3N0LWJ1aWxkCmNlcGguY29uZgo= testkey-2=aWIKTWFrZWZpbGUKbWFuCm91dApzcmMKVGVzdGluZwo=
-        rgw crypt require ssl = false
```

Comment:
we still want this config set everywhere so we can run the s3test cases for sse-c

Reply:
Sorry, I thought it was about the kms testing. Reverted.